puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman_bootdisk.yaml

```diff
@@ -1,5 +1,10 @@
 - project:
     name: foreman_bootdisk
     defaults: plugin
+    branch:
+      - develop:
```

Comment:
Thisd should be `master`, AFAICT.

Reply:
*How* did you know?!? :-) Indeed.

Reply:
This is the plugin's branch, `foreman_branch` is the branch name in the core repo, so `master` and `develop` need to get switched here now.

Reply:
Of course... Rebased.